docs/intro/build-your-first-app.md

```diff
@@ -0,0 +1,95 @@
+---
+slug: build-your-first-app
+title: Build your first app
+tags: []
+---
+
+# **Build your first app**
+
+This guide will walk you through creating and running your first FlutterFlow app. In this app, users can adjust the quantity of a product before adding it to their shopping cart.
+
+It’s very straightforward yet introduces you to some fundamental concepts in app development. Here's how it looks when completed:
+
+<iframe src="https://demo.arcade.software/YXXxpSfeOUgkt9JAOxUL?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="800"></iframe>
+
+> **What you'll learn**
+> - Creating layouts (add widgets)
+> - Adding interactivity to UI elements
+> - Handle app behavior in response to user interactions (manage state).
+> - Running your app
+
+#### The steps to build the app are as follows:
+
+1. [Clone or create project](#clone-project)
+2. [Building UI](#build-ui)
+3. [Customize style](#customize-style)
+4. [Add state](#add-state)
+5. [Run the app](#run-app)
+
+## 1. Clone or create project {#clone-project}
+
+To kick off your project, the first step is to [create a new project](#). However, to make things easier, we've already created a starter app for you. Simply open [this link](https://app.flutterflow.io/project/f-f-quick-start-app-umu392), click the '**Clone**' button, and the project will be instantly added to your account.
+
+![clone-project.png](../../static/img/clone-project.png)
+
+After cloning the project, you’ll see a page with product images and description and you’ll add a feature that allows users to update the product quantity.
+
+![zero-to-final.png](../../static/img/zero-to-final.png)
+
+## 2. Building UI {#build-ui}
+
+In this section, we will see how to build the user interface (UI) for this feature. This involves creating a layout and adding various widgets to our page.
+
+To build the UI:
+
+1. We'll begin by adding a [Container](#) widget. This will serve as the parent widget for our buttons and labels.
```

Comment:
"This will serve as" -> "This is the parent widget..." It serves as because it is.